docs/src/modules/utils/textToHash.js

```diff
@@ -18,7 +18,7 @@ export default function textToHash(text, unique = {}) {
         // eslint-disable-next-line no-useless-escape
         .replace(/[!@#\$%\^&\*\(\)=_\+\[\]{}`~;:'"\|,\.<>\/\?\s]+/g, '-')
         .replace(/-+/g, '-')
-        .replace(/-$/g, ''),
+        .replace(/^-|-$/g, ''),
```

Comment:
handle this problem: https://next.material-ui.com/customization/default-theme/#-material-ui-core-styles-vs-material-ui-styles